client/webpack.config.node.js

```diff
@@ -108,6 +114,13 @@ const webpackConfig = {
 				cacheIdentifier,
 				exclude: /node_modules\//,
 			} ),
+			TranspileConfig.loader( {
+				workerCount,
+				presets: [ require.resolve( '@automattic/calypso-build/babel/dependencies' ) ],
+				cacheDirectory: path.join( buildDir, '.babel-server-cache' ),
```

Comment:
I'm assuming we're consciously using the same cache strategy like on the babel loader above?

Reply:
That's right: the client build also [uses the same cache dir](https://github.com/Automattic/wp-calypso/blob/master/client/webpack.config.js#L210-L223) for both transpilation configs (app sources and package sources).

However, the cache dir resolves to `wp-calypso/build/.babel-server-cache`. @scinos is it desirable to move the server cache to the common `.cache` directory? The client compilation stores the Babel cache in `.cache/evergreen/babel-client`. We could use `.cache/babel-server` for the server build.

Reply:
+1 to moving it to `.cache/babel-server`

Reply:
Changing the cache dir in #45954. :+1:

Reply:
Thanks, Jarda 🙌